apps/etterlatte-beregning/src/main/kotlin/beregning/grunnlag/RegulerOverstyrt.kt

```diff
@@ -0,0 +1,101 @@
+        regulerOverstyrtKroneavrundet.eksekver(
+            grunnlag =
+                KonstantGrunnlag(
+                    RegulerManuellBeregningGrunnlag(
+                        manueltBeregnetBeloep =
+                            FaktumNode(
+                                verdi = Beregningstall(forrigeBeregningsgrunnlag.utbetaltBeloep.toInt()),
+                                Grunnlagsopplysning.Gjenny(Fagsaksystem.EY.navn, Tidspunkt.now()),
+                                beskrivelse = "Forrige manuelt overstyrte beregning",
+                            ),
+                        forrigeGrunnbeloep =
+                            FaktumNode(
+                                verdi = Beregningstall(forrigeGrunnbeloep.grunnbeloepPerMaaned),
+                                Grunnlagsopplysning.Gjenny(Fagsaksystem.EY.navn, Tidspunkt.now()),
+                                beskrivelse = "Forrige grunnbeløp brukt til å manuelt utregne beregning",
+                            ),
+                        nyttGrunnbeloep =
+                            FaktumNode(
+                                verdi = Beregningstall(nyttGrunnbeloep.grunnbeloepPerMaaned),
+                                Grunnlagsopplysning.Gjenny(Fagsaksystem.EY.navn, Tidspunkt.now()),
+                                beskrivelse = "Nytt grunbeløp beregnins skal reguleres etter",
+                            ),
+                    ),
+                ),
+            periode =
+                RegelPeriode(
+                    fraDato = reguleringsmaaned.minusMonths(1).atDay(1),
+                    tilDato = reguleringsmaaned.atEndOfMonth(),
+                ),
+        ).let {
+            when (it) {
+                is RegelkjoeringResultat.Suksess -> {
+                    it.periodiserteResultater.single()
+                }
+                is RegelkjoeringResultat.UgyldigPeriode ->
+                    throw RuntimeException("Ugyldig regler for periode: ${it.ugyldigeReglerForPeriode}")
+            }
+        }
+    return forrigeBeregningsgrunnlag.copy(
+        id = UUID.randomUUID(),
+        behandlingId = behandlingId,
+        datoFOM = reguleringsmaaned.atDay(1),
+        datoTOM = null,
+        utbetaltBeloep = resultat.resultat.verdi.toLong(),
+        kilde = Grunnlagsopplysning.automatiskSaksbehandler,
+        reguleringRegelresultat = objectMapper.valueToTree(resultat.resultat),
+        reguleringRegelVersjon = resultat.reglerVersjon,
+    )
+}
+
+private fun utledGrunbeloep(reguleringsmaaned: YearMonth) =
+    grunnbeloepUtenGrunnlag.eksekver(
+        grunnlag = KonstantGrunnlag(""),
+        periode =
+            RegelPeriode(
+                fraDato = reguleringsmaaned.minusYears(1).atDay(1),
+                tilDato = reguleringsmaaned.atEndOfMonth(),
+            ),
+    ).let { resultat ->
+        when (resultat) {
+            is RegelkjoeringResultat.Suksess -> {
+                assert(resultat.periodiserteResultater.size == 2)
```

Comment:
```suggestion
                check(resultat.periodiserteResultater.size == 2) {
                    "Fikk uventet antall perioder for utleding av grunnlag: ${resultat.periodiserteResultater.size}"
                }
```

så blir det kjørt uten --enable-assertions